dwitter/static/js/scrolling.js

```diff
@@ -1,34 +1,62 @@
 window.onload = function() {
-  var infinite = new Waypoint.Infinite({
+  var dweets = document.querySelectorAll('.dweet');
+  var dweetiframes = document.querySelectorAll('.dweetiframe');
+  var editor = document.querySelector('#editor');
+  var editoriframe = document.querySelector('#preview-iframe');
+  var oldCode = editor && editor.value;
+
+  [].forEach.call(dweetiframes, function(iframe) {
+    registerWaypoint(iframe);
+  });
+
+  [].forEach.call(dweets, function(dweet) {
+    registerOnKeyListener(dweet);
+    registerStatsClickListeners(dweet);
+  });
+
+  if (editor && editoriframe) {
+    // Update editor!
+    showCode(editoriframe, oldCode);
+    editor.addEventListener('keyup', function() {
+      if (editor.value === oldCode) {
+        return;
+      }
+      editor.size = Math.max(editor.value.length, 1);
+      showCode(editoriframe, editor.value);
+      oldCode = editor.value;
+    });
+  }
+
+  // eslint-disable-next-line no-new
+  new Waypoint.Infinite({
```

Comment:
The Airbnb ESLint config has a rule against invoking `new` just for the its side effects. In this case, it's easier to ignore the error than to change the Waypoint API 😄